### ISSUE TITLE
Inherit parameter defined at the path level

### ISSUE THREAD

**Issue by jmrtnz94** [alexcampana/ApiFirstMediatR#68]
Given the following definition, we expect the CarId property to be included in the GetCarByIdQuery.
```yaml
  /v1/car/{carId}:
    parameters:
      - in: path
        name: carId
        description: CarId of Car
        required: true
        schema:
          type: integer
          format: int32
    get:
      tags:
        - car
      summary: Get Car by `CarId`
      description: Use to retrieve an existing Car
      operationId: GetCarById
      responses:
        200:
          description: Success
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/CarDto"
```

For Reference:
URL: https://swagger.io/docs/specification/describing-parameters/
Section: Common Parameters for All Methods of a Path

**Comment by jmrtnz94** [comment]
Feature Request

**Comment by alexcampana** [comment]
Interesting, I would've assumed that OpenAPI.NET would've nicely parsed that out for us. Definitely a worthwhile addition and should be fairly easy to add. I'll try and knock that out in the next few days.

**Comment by alexcampana** [comment]
Available as of v1.0.11
